v3/test/helpers.go

```diff
@@ -35,7 +39,15 @@ import (
 // lint result is nil.
 //nolint:revive
 func TestLint(lintName string, testCertFilename string) *lint.LintResult {
-	return TestLintCert(lintName, ReadTestCert(testCertFilename))
+	return TestLintWithConfig(lintName, testCertFilename, "")
+}
+
+func TestLintWithConfig(lintName string, testCertFilename string, context string) *lint.LintResult {
+	ctx, err := lint.NewConfigFromString(context)
+	if err != nil {
+		panic(err)
+	}
```

Comment:
I think the more idiomatic thing to do would be to take `t *testing.T` as the first argument, declare this a `t.Helper()`, and then use `t.Fatal()` instead of directly `panic`ing.

But I recognize that that's not the pattern used by the other helpers in this file, so it's probably better to do it this way for now.

Reply:
Right, I wouldn't be opposed to that were it not for the massive pile of lint tests that have already been written using this interface (a perfunctory glance shows ~550) :sob: